src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -2702,12 +2665,9 @@ static TestResult DoTest(in Args args, int currentPid)
         Debug.Assert(SimpleRefPayLoad.NumPinned == SimpleRefPayLoad.NumUnpinned);
 #endif
 
+        Console.WriteLine($"Time Taken: {tEnd - tStart}ms");
```

Comment:
you might as well add a config to say if you want it to print to console or print to a file. so you don't do duplications like this